IPs/RSKIP176.md

```diff
@@ -0,0 +1,145 @@
+# Fast BTC Bridge
+|RSKIP          |           |
+| :------------ |:-------------|
+|**Title**      |Fast BTC Bridge |
+|**Created**    |15-SEP-20 |
+|**Author**     |GM |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Previous concepts
+
+#####Derived Federation
+The derived federation is the result of a process that will be carried out by creating a 32 bytes hash built from some provided derivation arguments and the active federation redeem script. The derivation arguments are values used to close the agreement between the Liquidity Provider and the user.
+
+## Abstract
+
+This RSKIP proposes a protocol change for the Bridge to provide a way for a contract to validate if a given BTC transaction sent funds to a derived Federation and to send those funds to that contract. This is part of a wider fast-bridge project that will allow a user to transfer BTC to RSK in a fast way, where a third party takes the risk to advance the payment in RBTC to the user.
+
+## Motivation
+
+Allowing users to transfer BTC to RSK immediately, without waiting for enforced peg-in confirmation blocks. This also generates a market between user who wants to peg-in faster and those whose who provide the liquidity to advance the payment and accept the risk of the reversal of the Bitcoin blockchain.
+
+## Specification
+
+The fast-bridge system connects users, Liquidity Providers (LP) and a certain Liquidity Bridge Contract (LBC).
+This RSKIP details the specification of consensus changes to achieve the main goal of the use case without limiting the actual implementation of the fast-bridge system.
+
+#### Bridge
+
+The bridge needs to expose a new method called **registerFastBridgeBtcTransaction**().
+
+#### registerFastBridgeBtcTransaction()
+
+**ABI Signature**
+
+        function registerFastBridgeBtcTransaction(bytes btcTxSerialized, uint256 height, bytes pmtSerialized, bytes32 derivationArgumentsHash, string userRefundBtcAddress, string LiquidityBridgeContractAddress, string LiquidityProviderBtcAddress, bool shouldTransferToContract) returns int executionStatus
+
+
+**Derivation Arguments**
+
+The derivation arguments are values used to close the agreement between the Liquidity Provider and the user. Some of these parameters are the RSK address where funds will be received, the BTC refund address, the Liquidity Bridge Contract address, the value to transfer, the amount of BTC blocks that the Liquidity Provider should wait before advancing the funds, the Liquidity Provider BTC refund address, the derived Federation address, and some other values that are domain of the Liquidity Provider.
```

Comment:
First you call them arguments, but then "parameters". Let's stick to "arguments"